ecms_base/config/install/user.role.drupal_administrator.yml

```diff
@@ -0,0 +1,37 @@
+langcode: en
+status: true
+dependencies: {  }
+id: drupal_administrator
+label: 'Drupal Administrator'
+weight: 3
+is_admin: null
```

Comment:
We should probably mark this role as an admin, so it inherits all permissions by default. Maybe we should rename the existing `administrator` role to this instead?

Reply:
Got it, going to update to match the roles & publishing workflows document.